### PR TITLE
Examples enhancer

### PR DESCRIPTION
**Pull request by markstur**

### CODE REVIEW

File: examples/cli/enhancer/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+```
+
+### Example 1: HIDDEN inside the license match
+
+We log the matching block of license text. Notice here that, keywords and copyright are INSIDE the matched license text and so they are not logged separately.
+
+```shell
+go run ../../.. --file testdata/ANTLR-PD.txt --copyrights --keywords
+go run ../../.. --file testdata/0BSD.txt --copyrights --keywords
```
Comment (pritidesai) [inline examples/cli/enhancer/README.md:31]: how about adding the result of this `go run`?

**Comment by pritidesai** [comment]
the changes in https://github.com/CycloneDX/license-scanner/pull/16/commits/cacbb32097b6a42aac06e3c695f501ea437f268e looks good to me except the result of the commands which will clearly demonstrate the usage. I can merge this PR and we can add the command output in a separate PR.

**Comment by markstur** [comment]
Sounds good to me.  I was inconsistent on showing output because I wrote this for a demo and had mixed feelings about the extra long guide vs live demo. But that's why it's great to have reviewer suggestions.  Probably best to have output added in a separate PR.

**Review by pritidesai — APPROVED** [review]

**Comment by pritidesai** [comment]
sounds good, thank you @markstur 👍

**Pull request merged**